back_end/README.md

```diff
@@ -1,21 +1,83 @@
 # Backend Project
 
-The following project is to build a simple service that leverages some of the technologies that we use as part of our reference architecture.
+The following project is to build a simple service leveraging API best practices. This should take you 4 hours to complete, but you are free to spend more time on it.
 
-If any special requirements have been conveyed to you from your interview team, they supercede the following:
+If any special requirements have been conveyed to you from your interview team, they supersede the following:
+
+## API Specification
+
+We want to create an API that allow us to handle articles that could be shown in many clients like mobile apps, websites, text to speech apps, etc.
+ 
+ For this challenge assume that the writers are already store in the application database. You don't need to create a registration endpoint.
+ 
+## Actors 
+
+### Visitor
+Is any person that wants to read the articles published in the API. It doesn't have to authenticate herself to do it. 
+
+### Writer
+
+Is a person that have the permission to create new articles in our API. It has to authenticate herself in order to create, edit and publish articles.
+
+## Entities
+
+This is a list of entities that need to be on the API.  
+
+### Article
+
+An article can be on two possible states. The first one is **drafted** that means that only the writer can see it. The second one is "published" that means that everyone can see it. 
+
+Article attributes: 
+
+- title
+- body
+- published date
+- created_at
+- state
+
+### Writer
+
+Writer attributes: 
+
+- first name
+- last name
+
+## User Stories 
```

Comment:
I like this a lot. Writing this as user stories makes them easier to write better tests.

How much time the candidate have to complete this challenge?